### ISSUE TITLE
Perhaps rule 3 should ignore comments?

### ISSUE THREAD

**Issue by axelarge** [oakmac/standard-clojure-style-js#109]
At the risk of introducing too much special casing, have you considered skipping over comments when determining if [rule 3](https://github.com/clj-commons/formatter/issues/9#issuecomment-446167649) should be applied?

Consider this code:
```clojure
(or condition1
         ;; comment
    condition2
    condition3)
```
Because the comment is used to check if rule 3 should be triggered, it gets reformatted into:
```clojure
(or condition1
  ;; comment
  condition2
  condition3)
```

While I understand the reasoning, personally I expected it to result in
```clojure
(or condition1
    ;; comment
    condition2
    condition3)
```

What do you think?
